src/xGadgetron/pGadgetron/tests/test1.py

```diff
@@ -172,9 +172,14 @@ def test_main(rec=False, verb=False, throw=True):
     print('%f is 0.0' % d)
     test.check_if_equal(1, d < 1e-6)
     # test on fill with scalar
-    for n in (2,2.0,numpy.int32(2),numpy.int64(2),numpy.float128(2),numpy.complex(2)):
+    twos = (2,2.0,numpy.int32(2),numpy.int64(2),numpy.complex(2))
+    try:
+        twos += numpy.float128(2)
+    except:
+        pass
+    for n in twos:
         complex_images.fill(n)
-        test.check_if_zero_within_tolerance((complex_images-2).norm())
+        test.check_if_equal(0, (complex_images-2).norm())
```

Comment:
why replace with `check_if_equal`? You can never know with numerical calculations.

Reply:
`complex_images` is filled with 2s, which are represented exactly

Reply:
fair enough, but suppose someone changes it to some other value (admittedly a bit hard to get a non-exact representation with integers, unless it'd be really big). In any case, it seems best practice to me to check real numbers with tolerance.

Reply:
nobody is supposed to change anything in our regression tests without giving a sound reason, and I see no such reason for changing 2 to any other value - the whole point of this test is to ensure `fill` can handle scalars

Reply:
There is no reason to change it from 2, but I see no good reason to have a test that is stronger than what will work for most scalars. I find it misleading. I'd prefer that you revert it.

Reply:
ok